src/terrain/routes/schemas/dashboard_aggregator.clj

```diff
@@ -129,10 +87,10 @@
 
 (defschema DashboardAggregatedAnalyses
   {(optional-key :recent)
-   (describe (maybe [DashboardAnalysis]) "Analyses recent launched by the user")
+   (describe (maybe analyses-listing/AnalysisList) "Analyses recent launched by the user")
 
    (optional-key :running)
-   (describe (maybe [DashboardAnalysis]) "Analyses currently running for the user")})
+   (describe (maybe analyses-listing/AnalysisList) "Analyses currently running for the user")})
```

Comment:
If we get rid of the wrapping `{analyses: ...}` object in cyverse-de/dashboard-aggregator#9, then these would be `[analyses-listing/Analysis]` instead.

For example:
```suggestion
   (describe (maybe [analyses-listing/Analysis]) "Analyses currently running for the user")})
```